src/audio/crossover/crossover_ipc3.c

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Google LLC. All rights reserved.
+//
+// Author: Sebastiano Carlucci <scarlucci@google.com>
+
+#include <sof/audio/module_adapter/module/generic.h>
+#include <sof/audio/buffer.h>
+#include <sof/audio/component.h>
+#include <sof/audio/data_blob.h>
+#include <sof/audio/format.h>
+#include <sof/audio/pipeline.h>
+#include <sof/audio/ipc-config.h>
+#include <sof/common.h>
+#include <rtos/panic.h>
+#include <sof/ipc/msg.h>
+#include <rtos/alloc.h>
+#include <rtos/init.h>
+#include <sof/lib/memory.h>
+#include <sof/lib/uuid.h>
+#include <sof/math/iir_df2t.h>
+#include <sof/list.h>
+#include <sof/platform.h>
+#include <rtos/string.h>
+#include <sof/trace/trace.h>
+#include <sof/ut.h>
+#include <ipc/control.h>
+#include <ipc/stream.h>
+#include <ipc/topology.h>
+#include <user/trace.h>
+#include <user/eq.h>
+#include <errno.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <limits.h>
+
+#include "crossover_algorithm.h"
+#include "crossover_user.h"
+#include "crossover.h"
```

Comment:
30+ headers for 70 lines of code... sorry, cannot believe that

Reply:
forget to remove this part, sorry.